src/views/v2/month/calendar-body/day.php

```diff
@@ -76,7 +79,16 @@
 	>
 		<h3 class="tribe-events-calendar-month__day-date tribe-common-h6 tribe-common-h--alt">
 			<span class="tribe-common-a11y-visual-hide">
-				<?php echo esc_html( sprintf( _n( '%s event', '%s events', $day['found_events'], 'the-events-calendar' ), number_format_i18n( $day['found_events'] ) ) ); ?>,
+				<?php
+				echo esc_html(
+					sprintf(
+						_n( '%1$s %2$s', '%1$s %3$s', $day['found_events'], 'the-events-calendar' ),
```

Comment:
* :warning: Mismatched placeholders is probably an error
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 27659638292f1d511a260755fcdc42e1</sub>